tests/run.sh

```diff
@@ -1,28 +1,44 @@
 #!/usr/bin/env bash
 
 # Common variables shared among all tests
-export NEST="cargo run --bin=nest"
+export CARGO="env cargo"
+export NEST_SERVER="/tmp/nest-server"
+export PYTHONPATH=$(dirname "$0")
 
 # Colors
 export RED="\033[1;31m"
 export GREEN="\033[1;32m"
 export RESET="\033[0m"
 
-declare -i nb_tests=1
+if [ ! -d $NEST_SERVER ]; then
+    echo "Cloning latest nest-server..."
+    git clone https://github.com/raven-os/nest-server $NEST_SERVER
+elif [ ! -e $NEST_SERVER ]; then
+    echo "$NEST_SERVER already exists and is not a directory, aborting."
+    exit 1
+fi
+
+pushd $NEST_SERVER
+$CARGO build
+popd
+
```

Comment:
You should test if `/tmp/chroot` already exists here, to make sure you won't erase it later without permission. If it is the case, warn the user and suggest {him,her} to remove the directory {him,her}self.